tests/unittests/core/bad_config1.txt

```diff
@@ -0,0 +1,31 @@
+lalala~uniform(1, 3, shape=(100, 3))
```

Comment:
Personally would rename the files such that their name convey what the problem is.